### PR TITLE
Anirudh/update to 15

### PR DESCRIPTION
**Pull request by anirudh-satish**
Upgraging types-lf-ui-components and lf-ui-components-services to 15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "@angular/elements": "^13.2.1",
     "@laserfiche/lf-js-utils": "^4.0.4",
     "@laserfiche/lf-repository-api-client": "^1.1.3",
```
Comment (alexgomezlf) [inline package.json:8]: update version property in package.json
then run 'npm i' to update property in package-lock.json

File: src/config.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const config = {
-  REDIRECT_URI: 'REPLACE_WITH_YOUR_REDIRECT_URI', // i.e http://localhost:3000, https://serverName/lf-sample/index.html
-  CLIENT_ID: 'REPLACE_WITH_YOUR_CLIENT_ID',
+  REDIRECT_URI: 'http://localhost/lf-sample-app-react', // i.e http://localhost:3000, https://serverName/lf-sample/index.html
```
Comment (alexgomezlf) [inline src/config.ts:2]: revert this
try not to commit this file at all

Comment (anirudh-satish) [reply src/config.ts:2]: oops, forgot to revert on this repo

**Review by alexgomezlf — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
```
Comment (alexgomezlf) [inline package.json:5]: update version for URL in readme (lfxstatic)

**Review by alexgomezlf — APPROVED** [review]

**Pull request merged**